ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py

```diff
@@ -1307,3 +1307,39 @@ def test_tensorflow_crelu(
         features=x[0],
         axis=axis,
     )
+
+
+# conv_transpose
+@handle_frontend_test(
+    fn_tree="tensorflow.nn.conv_transpose",
+        x_f_d_df=_x_and_filters(
+        dtypes=helpers.get_dtypes("float", full=False),
+        data_format=st.sampled_from(["channels_last"]),
```

Comment:
You are getting an error inside the helper function `_x_and_filters`, and that is because you are not generating any `x_shape` parameter inside the function. However you are passing it inside `array_values`. The `x_shape` generation is not happening because you are passing the wrong `data_format`. There is no such thing as `channels_last`. If you look clearly inside the helper, you'll see that the `data_format` is either of these - `[NWC, NHWC, NDHWC`]

Reply:
Hi @anurag629, please use the `x_and_filters` helper only. Just make the changes i have suggested above

Reply:
> You are getting an error inside the helper function `_x_and_filters`, and that is because you are not generating any `x_shape` parameter inside the function. However you are passing it inside `array_values`. The `x_shape` generation is not happening because you are passing the wrong `data_format`. There is no such thing as `channels_last`. If you look clearly inside the helper, you'll see that the `data_format` is either of these - `[NWC, NHWC, NDHWC`]

Hi @anurag629 please use `x_and_filters` only. Just make the changes i have suggested above.